aten/src/ATen/native/ReplicationPadding.cpp

```diff
@@ -220,6 +161,11 @@ TORCH_META_FUNC(replication_pad3d) (
   int64_t oheight = iheight + ptop + pbottom;
   int64_t owidth  = iwidth + pleft + pright;
 
+  TORCH_CHECK(owidth >= 1 || oheight >= 1 || odepth >= 1,
```

Comment:
Is this constraint new or something we just failed to check so far?

Reply:
This part is a copy paste from https://github.com/pytorch/pytorch/pull/102253/files/e854341982367927e1792b37b172bba7d1f0142d#diff-1e3e579837dd1291362434c7935fbe62601e5b3511820a7e6507fe4d8fbd879bL178

the error message is identical to previous.